project/plugins.sbt

```diff
@@ -1,22 +1,16 @@
-addSbtPlugin("ch.epfl.scala"                     % "sbt-bloop"                     % "1.5.17")
-addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"                  % "0.12.1")
-addSbtPlugin("com.eed3si9n"                      % "sbt-buildinfo"                 % "0.12.0")
-addSbtPlugin("com.eed3si9n"                      % "sbt-unidoc"                    % "0.4.3")
-addSbtPlugin("com.github.sbt"                    % "sbt-ci-release"                % "1.5.10")
-addSbtPlugin("com.github.cb372"                  % "sbt-explicit-dependencies"     % "0.2.16")
-addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings"              % "3.0.2")
-addSbtPlugin("com.typesafe"                      % "sbt-mima-plugin"               % "1.1.0")
-addSbtPlugin("de.heikoseeberger"                 % "sbt-header"                    % "5.7.0")
-addSbtPlugin("org.portable-scala"                % "sbt-scala-native-crossproject" % "1.2.0")
-addSbtPlugin("org.portable-scala"                % "sbt-scalajs-crossproject"      % "1.2.0")
-addSbtPlugin("org.scala-js"                      % "sbt-scalajs"                   % "1.16.0")
-addSbtPlugin("org.scala-native"                  % "sbt-scala-native"              % "0.4.16")
-addSbtPlugin("org.scalameta"                     % "sbt-mdoc"                      % "2.3.7")
-addSbtPlugin("org.scalameta"                     % "sbt-scalafmt"                  % "2.4.6")
-addSbtPlugin("pl.project13.scala"                % "sbt-jcstress"                  % "0.2.0")
-addSbtPlugin("pl.project13.scala"                % "sbt-jmh"                       % "0.4.4")
-addSbtPlugin("dev.zio"                           % "zio-sbt-website"               % "0.3.1")
-
-libraryDependencies += "org.snakeyaml" % "snakeyaml-engine" % "2.6"
```

Comment:
was unused? or is now a transitive dependency?

Reply:
It was used to extract Scale versions from the `ci.yml` definition. There is quite some difference on how different zio-projects are being configured. The `BuildHelper.scala` has a lot if different implementations. I think the Scala versions should just be part of the build files and not extracted from ci.yml. Most projects do this and it prevent us from requiring this dependency in the sbt tasks.